addchain/main.go

```diff
@@ -0,0 +1,117 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/viper"
+)
+
+func main() {
+	chainType := flag.String("chain_type", "", "Type of chain to add. Must be 'standard' or 'frontier'.")
+	help := flag.Bool("help", false, "Show help information")
+	flag.Parse()
+
+	if *help {
+		showUsage()
+		return
+	}
+
+	superchainLevel, err := getSuperchainLevel(*chainType)
+	if err != nil {
+		fmt.Println("Failed to get superchain level:", err)
+		showUsage()
+		os.Exit(1)
+	}
+
+	// Get the current script's directory
+	scriptDir, err := os.Getwd()
+	if err != nil {
+		fmt.Println("Error getting current directory:", err)
+		os.Exit(1)
+	}
+	superchainRepoPath := filepath.Dir(scriptDir) // Parent directory
+
+	// Load environment variables
+	viper.SetConfigName(".env")             // name of config file (without extension)
```

Comment:
Not sure if the `viper` package is the best option to pull in env vars. Will look at other options

Reply:
This is working so will leave as is. Can replace in a future PR if we want